src/components/common/DragAndDrop/DragAndDrop.jsx

```diff
@@ -33,30 +33,31 @@ const DragAndDrop = ({onFilesSelected, updateUploadedFiles}) => {
     const droppedFiles = e.dataTransfer.files;
     if (droppedFiles && droppedFiles[0]) {
       const newFiles = Array.from(droppedFiles);
-      setFiles((prevFiles) => [...prevFiles, ...newFiles]);
-      console.log("DROPPED", files);
-      memoizedUpdateUploadedFiles(newFiles);
+      // setFiles((prevFiles) => [...prevFiles, ...newFiles]);
```

Comment:
Removed unnecessary state update and function call.

Note that logging here captures a snapshot of the state at the same instant it is being updated, so the previous state (an empty array if no images have been uploaded yet) will be logged.